src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const MIN_GB = 0;
```

Comment:
just a nit: I think these constants could be more verbose (to avoid the need to comment them).  For example:

`INIT_GB` -> `DEFAULT_LEDGER_STORAGE_GB`
`MAX_GB` -> `MAX_LEDGER_STORAGE_GB`
`DEFAULT_LAMPORTS` -> `DEFAULT_MINIMUM_DEPOSIT_LAMPORTS`
`CLUSTER_UPDATE_TIMEOUT` -> perfect!

Reply:
done